src/main/java/uk/gov/hmcts/reform/coh/controller/AnswerController.java

```diff
@@ -39,31 +36,27 @@
 public class AnswerController {
     private static final Logger log = LoggerFactory.getLogger(AnswerController.class);
 
+    @Autowired
     private AnswerService answerService;
 
+    @Autowired
     private AnswerStateService answerStateService;
 
+    @Autowired
     private QuestionService questionService;
 
+    @Autowired
+    private QuestionStateService questionStateService;
+
+    @Autowired
     private OnlineHearingService onlineHearingService;
 
+    @Autowired
     private AnswersReceivedTask answersReceivedTask;
 
     @Autowired
     private SessionEventService sessionEventService;
 
```

Comment:
I just got tired of the updating the constructor every time I needed to auto wire a service :-)